src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -87,28 +97,32 @@ export class SortableTableComponent implements OnInit, OnChanges {
     if (!this.columnToSortBy) {
       return;
     }
-    const columnIndex: number = this.columns.findIndex(
-        (column: ColumnData) => column.header === this.columnToSortBy);
+    const columnIndex: number = this.columns.findIndex((column: ColumnData) => column.header === this.columnToSortBy);
     if (columnIndex < 0) {
       return;
     }
     const sortBy: SortBy | undefined = this.columns[columnIndex].sortBy;
     if (!sortBy) {
       return;
     }
-
+    this.sortEvent.emit({ sortBy, sortOrder: this.sortOrder });
```

Comment:
`sortable-table` emits both the column to sort by and the sorting order. This is necessary to sync the `tableRowsModels` and `rowsData` together.